pkg/capabilities/mercury/mercury.go

```diff
@@ -30,6 +30,11 @@ func (id FeedID) Bytes() [FeedIDBytesLen]byte {
 	return [FeedIDBytesLen]byte(b)
 }
 
+// NOTE: Torn about this since we already have JSON Schema validation
+// on mercury trigger side.
+//
+// feeds_aggregator uses this, but on the previous round outcome
+// which should have already clean data?
```

Comment:
@bolekk could you help me confirm this?

Reply:
I'm OK replacing with your validator.

Reply:
Leaving it in for now. Validators should be left to capabilities only. @bolekk I'd only be comfortable with removing this if the data coming into `Aggregate` is guaranteed to be validated. Then we dont need this check at all.